src/chrome/content/rules/Rubicon-Project.xml

```diff
@@ -15,53 +15,134 @@
+		- garagestaging.rubiconproject.com
+		- go.rubiconproject.com
+		- investor.rubiconproject.com
+		- labs2.rubiconproject.com
+		- messages.rubiconproject.com
+		- assets.rfm.rubiconproject.com
+		- rubibid.rubiconproject.com
+		- stats.rubiconproject.com
+		- status.rubiconproject.com
+		- tap-cdn.rubiconproject.com
+	
+	Non-200 status code:
+		- tap2-cdn.rubiconproject.com
 
 -->
 <ruleset name="Rubicon Project.com">
 
-	<!--	Direct rewrites:
-				-->
-	<target host="ads.rubiconproject.com" />
+	<target host="rubiconproject.com" />
+	<target host="www.rubiconproject.com" />
+	
+	<target host="49bc.rubiconproject.com" />
 	<target host="anvil.rubiconproject.com" />
-	<target host="arsenal.rubiconproject.com" />
-	<target host="beacon-apac-hkg1.rubiconproject.com" />
+	<target host="ads.aws.rubiconproject.com" />
+	<target host="prebid.aws.rubiconproject.com" />
+	<target host="sneezy.aws.rubiconproject.com" />
+	<target host="stats.aws.rubiconproject.com" />
+	<target host="usync.aws.rubiconproject.com" />
+	<target host="yahoo.aws.rubiconproject.com" />
+	<target host="assets-rfm.rubiconproject.com" />
+	<target host="beacon.rubiconproject.com" />
+	<target host="beacon-eu-ams3.rubiconproject.com" />
 	<target host="beacon-eu2.rubiconproject.com" />
+	<target host="beacon-apac-nrt1.rubiconproject.com" />
+	<target host="beacon-apac-hkg1.rubiconproject.com" />
 	<target host="beacon-us-east.rubiconproject.com" />
-	<target host="beacon.rubiconproject.com" />
+	<target host="beacon-nf.rubiconproject.com" />
+	<target host="beacon-us-iad2.rubiconproject.com" />
+	<target host="beacon-us-iad3.rubiconproject.com" />
+	<target host="beacon-us-sjc1.rubiconproject.com" />
+	<target host="beacon-us-west.rubiconproject.com" />
+	<target host="buyer.rubiconproject.com" />
+	<target host="chango.rubiconproject.com" />
+	<target host="connect.rubiconproject.com" />
+	<target host="connect-api.rubiconproject.com" />
+	<target host="ebsdrdmz.cs.rubiconproject.com" />
+	<target host="ebsproddmz.cs.rubiconproject.com" />
 	<target host="demand.rubiconproject.com" />
-	<target host="fastlane.rubiconproject.com" />
+	<target host="dev.rubiconproject.com" />
+	<target host="dspdash.rubiconproject.com" />
+	<target host="eus.rubiconproject.com" />
+	<target host="exapi-apac.rubiconproject.com" />
+	<target host="exapi-eu.rubiconproject.com" />
+	<target host="exapi-us-east.rubiconproject.com" />
+	<target host="exapi-us-west.rubiconproject.com" />
+	<target host="finance.rubiconproject.com" />
+	<target host="flapi1.rubiconproject.com" />
+	<target host="flds.rubiconproject.com" />
+	<target host="idm.rubiconproject.com" />
+	<target host="integration.rubiconproject.com" />
+	<target host="kb.rubiconproject.com" />
+	<target host="labs1.rubiconproject.com" />
 	<target host="login.rubiconproject.com" />
-	<target host="optimized-by.rubiconproject.com" />
-	<target host="pixel.rubiconproject.com" />
-	<target host="revv-static.rubiconproject.com" />
```

Comment:
All three WFM.
You may find test urls with the help of search engines (e.g. `http://pixel.rubiconproject.com/tap.php`)